apis/management.cattle.io/v3/multi_cluster_app.go

```diff
@@ -39,3 +40,11 @@ type Answer struct {
 	ClusterName string            `json:"clusterName,omitempty" norman:"type=reference[cluster]"`
 	Values      map[string]string `json:"values,omitempty" norman:"required"`
 }
+
+type Member struct {
+	UserName           string `json:"userName,omitempty" norman:"type=reference[user]"`
```

Comment:
talked to @cjellick about the necessity to have both fields - *name and *principalName. Looks like we should be fine with just having *principalName